courses/free-courses-pt_BR.md

```diff
@@ -296,6 +297,12 @@
 * [Selenium com Python](https://www.youtube.com/playlist?list=PLOQgLBuj2-3LqnMYKZZgzeC7CKCPF375B) - Eduardo Mendes (YouTube)
 
 
+### R
+
+* [Curso de R com R Studio](https://www.youtube.com/watch?v=2mS-SzVWPZQ&list=PLzWDDw1w8cTS4i_B49WOWtjngjcMqTruG) - Escola de Inteligência Artificial
+* [Curso de R para iniciantes](https://www.youtube.com/watch?v=tfPsmDzS74c&list=PLyqOvdQmGdTQ5dE6hSD7ZGBu8bud70wYf) - Didática Tech
```

Comment:
- Use titles as same as provided by original source
- We prefer YT playlist URLs

```suggestion
* [Curso R para Iniciantes](https://www.youtube.com/playlist?list=PLyqOvdQmGdTQ5dE6hSD7ZGBu8bud70wYf) - Didática Tech
```